packages/editor/src/client/blocks/richTextBlock/toolbar/linkMenu.tsx

```diff
@@ -0,0 +1,168 @@
+import React, {useState, useEffect, useContext} from 'react'
+import {useTranslation} from 'react-i18next'
+import {Button, Form, FormGroup, ControlLabel, FormControl, ButtonToolbar} from 'rsuite'
+import {Transforms, Range, Editor} from 'slate'
+import {useSlate} from 'slate-react'
+import {WepublishEditor} from '../editor/wepublishEditor'
+import {InlineFormat} from '../editor/formats'
+import {SubMenuContext} from '../../../atoms/toolbar'
+import axios from 'axios'
+
+export function LinkMenu() {
+  const editor = useSlate()
+
+  const {closeMenu} = useContext(SubMenuContext)
+
+  const [title, setTitle] = useState('')
+  const [url, setURL] = useState('')
+
+  const [selection, setSelection] = useState<Range | null>(null)
+
+  const [isValidURL, setIsValidURL] = useState(false)
+  const isDisabled = !isValidURL
```

Comment:
The button shouldn't be disabled if we can't validate the url. It might be that our validation failed or that the URL will only be active at a later date.

Reply:
Should it be disabled if it doesn't pass the regex, or just remove the disabled part completely so there's always an option to add invalid links?

Reply:
@penk85 I think it should be disable if there is no value in the link or text input. But if the link seems to be invalid it should only show the warning but allow the input.